### ISSUE TITLE
Can not convert png:"convert: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/509."

### ISSUE THREAD

**Issue by sfon** [ImageMagick/ImageMagick#675]
`./configure` and `make` and `make install` they are running OK, but when i `make check`, there 2 FAIL:
FAIL: Magick++/demo/demos.tap 6
FAIL: tests/wandtest.tap 1
and `convert logo: logo.gif` running OK, but when i convert a png format pic from website, it failed:
$ /usr/local/bin/convert -008.png 008.jpeg
convert: no decode delegate for this image format 'PNG' @ error/constitute.c/ReadImage/509.
convert: no images defined '008.jpeg' @ error/convert.c/ConvertImageCommand/3275.
i do not why, would you help me?

**Comment by bdheeman** [comment]
Seems you attempted to locally build the IM from source yourself.
If yes, you seems to have not build and, or installed requisite dependencies e,g, libpng in particularly in your case.

Which version of IM you're trying build? and on which platform and, or distribution are also important tell to get quick and proper response.

**Comment by clairehe** [comment]
@sfon @bdheeman Hi, got the same error when trying to install IM locally Ubuntu 16.04. 
I configured it with `./configure --prefix=/my/local/home`. libpng=1.6.17 is installed in  /local/anaconda2/lib. Seems IM is unable to link it. By adding anaconda/lib into $LD_LIBRARY_PATH doesn't help.
Any tips?

**Comment by clairehe** [comment]
@sfon hi but the problem has not been solved yet

**Comment by sfon** [comment]
More than two months ago, I tried using the source code installation ImageMagick and found the error. Just two days ago I installed ImageMagick the latest version of the RPM package, it worked very well, so I closed the problem. 
I do not know the cause of this problem, my operating system version is:
`Linux CentOS 3.10.0-693.2.2.el7.x86_64 #1 SMP Tue Sep 12 22:26:13 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
Thanks for your work, I think this software is great.
